src/cronParser.ts

```diff
@@ -40,6 +40,25 @@ export class CronParser {
     // split on one or more spaces
     let parsed: string[] = expression.trim().split(/[ ]+/);
 
+    // sort elements with array of numbers
+    for (let i = 0; i < parsed.length; i++) {
+      if (parsed[i].includes(",")) {
+        const arrayElement = parsed[i]
+          .split(",")
+          .map((item) => item.trim())
+          .filter((item) => item !== "")
+          .map((item) => (!isNaN(Number(item)) ? Number(item) : item))
+          .filter((item) => item !== null && item !== "");
+
+        if (arrayElement.length === 0) {
+          arrayElement.push("*");
+        }
```

Comment:
What is this for?

Reply:
To support the following:
`it("dayOfWeek specified as comma", function () {
      assert.equal(new CronParser("*/5 * * * * ,").parse()[5], "*");
    });`

Reply:
Is that a valid syntax for a cron?

Reply:
`*/5 * * * * ,` is an invalid syntax, this is why it parsed to `*/5 * * * * *` which is a valid syntax.
anyhow, it was an existing test that I didn't want to break.
My challenge was to add this fix while keeping all the 212 (+1 the one I added) running.